addons/binding/org.openhab.binding.sleepiq/src/main/java/org/openhab/binding/sleepiq/config/SleepIQCloudConfiguration.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.sleepiq.config;
+
+/**
+ * Configuration class for the SleepIQ cloud.
+ *
+ * @author Gregory Moyer - Initial contribution
+ */
+public class SleepIQCloudConfiguration {
+
+    public static final String USERNAME = "username";
+    public static final String PASSWORD = "password";
+    public static final String POLLING_INTERVAL = "pollingInterval";
+    public static final String LOGGING = "logging";
```

Comment:
Sorry, but I agree with @martinvw's concerns as the Thing configuration is really meant for "end users" and thus should not be about developer / logging options - this I would say, checking `logger.isDebugEnabled()` is the way to go. If there are technical constraints why the log level cannot be changed during runtime, so be it; the user will require a restart of his runtime in that case.

Anyhow, just a word of warning: Jersey is not made for OSGi and it actually behaves very nastily. Registering resources/filters with Jersey at runtime can lead to all kind of buggy behavior as usually the whole Jersey stack needs to be fully restarted.